docs/tutorial.rst

```diff
@@ -1,4 +1,620 @@
 Tutorial
 ========
 
-*Coming soon!*
+
+.. note::
+
+   This document starts out with a fairly simple application code
+   and proceeds by building on it.
+   Therefore, it would be helpful to the reader
+   to code along and try out the various stages of the application.
+   In this manner, completing it should take about an hour.
+
+
+In this tutorial, we are going to develop an application
+that will convert a given time (and date) between two time zones.
+The user will enter a date and a time,
+and select two time zones from a list of all available time zones,
+one for the source location and one for the destination location.
+
+Before we start, a note about time zones:
```

Comment:
I also think it would be a good idea to include a small screenshot of the finished application somewhere in the first section here so people know what the finished product looks like.

Reply:
Yes, done.